src/styles/blocks/catalog.scss

```
@@ -17,9 +18,16 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    cursor: pointer;
+    transition: all 300ms;
```

Better do not use all value. Use only those you used
```suggestion
    transition: color 0.3s, transform 0.3s;
```